Godot_test/ClientScripts/GameLevel.cs

```diff
@@ -22,6 +22,7 @@ public partial class GameLevel : Node
 	string pathToEnemies = "res://Sprites/LivingEntities/Enemies/";
 	public static List<GodotGame.Entities.Enemy> enemyList = new();
 	public static Dictionary<Roguelike.Tiles.Interactive.InteractiveTile, Node> interactiveList = new();
```

Comment:
That member shall not be static as we do not want to share items with other level instances.